dietpi/dietpi-software

```diff
@@ -15272,7 +15222,7 @@ _EOF_
 				for i in ${!aSOFTWARE_INSTALL_STATE[@]}
 				do
 
-					if (( ${aSOFTWARE_AVAIL_G_HW_MODEL[$i,$G_HW_MODEL]:=1} && ${aSOFTWARE_AVAIL_G_HW_ARCH[$i,$G_HW_ARCH]:=1} && ${aSOFTWARE_AVAIL_G_DISTRO[$i,$G_DISTRO]:=1} )) &&
+					if (( ${aSOFTWARE_AVAIL_G_HW_MODEL[$i,$G_HW_MODEL]:-1} && ${aSOFTWARE_AVAIL_G_HW_ARCH[$i,$G_HW_ARCH]:-1} && ${aSOFTWARE_AVAIL_G_DISTRO[$i,$G_DISTRO]:-1} )) &&
```

Comment:
@Fourdee 
I used assigning here for the reason that those array entries are not pre-defined, by used afterwards. I think empty entries are handled later as well, when it comes to an install, but using `:=1` avoids the need to re-set/assign the valze again every time it is checked.

Reply:
@MichaIng 

> but using :=1 avoids the need to re-set/assign the value again every time it is checked.

Ah apologies, in my testing, ```:-1``` and ```:=1``` have the same effect, both do the same task of static assign?